### PR TITLE
Initialize gradle and GitHub Actions

### PR DESCRIPTION
**Pull request by manuandru**

### CODE REVIEW

**Comment by manuandru** [comment]
Initialised gradle for building with java and automatize the workflow with GitHub Actions

File: src/main/java/Main.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+public class Main {
+
```
Comment (kelvin-olaiya) [inline src/main/java/Main.java:3]: Please provide Javadoc!

File: build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+plugins {
+    java
+}
```
Comment (kelvin-olaiya) [inline build.gradle.kts:3]: We would like to run the application, consider using the application plugin

**Review by kelvin-olaiya — CHANGES_REQUESTED** [review]
Nice PR! 
I've noticed there's conflicts, please fix them before looking the other comments!

File: .gitignore [hunk 3]
```diff
@@ -1,5 +1,11 @@
...
+.gradle/
+
+build/
+=======
 .gradle
 build
 bin
 .idea
 .vscode
+>>>>>>> 57358dd809ce1789d8fdd7697e2ea57e4fe6d9a8
```
Comment (kelvin-olaiya) [inline .gitignore:11]: WHAT IS THIS!?

**Review by kelvin-olaiya — CHANGES_REQUESTED** [review]

**Review by kelvin-olaiya — APPROVED** [review]
Looks good!

**Pull request merged**